custom-recipes/warpscript/src/com/dataiku/dip/plugins/warpscript/WarpScriptRecipe.java

```diff
@@ -93,16 +93,17 @@ public void run(ExecutionContext context, BackendClient client) throws Exception
             for (Object item : resultList) {
                 if (!(item instanceof GeoTimeSerie)) {
                     throw new CodedException(RecipeCodes.ERR_RECIPE_GENERIC_ERROR,
-                            "Resulting List must contain only GeoTimeSeries objects after execution");
+                            "Found a " + item.getClass().getName() + " in the resulting List, only GeoTimeSeries " +
+                                    "objects are allowed");
                 }
 
                 resultGts.add((GeoTimeSerie) item);
             }
         } else if (poppedObject instanceof GeoTimeSerie) {
             resultGts = Collections.singletonList((GeoTimeSerie) poppedObject);
         } else {
-            throw new CodedException(RecipeCodes.ERR_RECIPE_GENERIC_ERROR,
-                    "Element on the stack is neither a List nor a GeoTimeSeries object");
+            throw new CodedException(RecipeCodes.ERR_RECIPE_GENERIC_ERROR, "Element on the stack is a " +
+                    poppedObject.getClass().getName() + ", it must be either a List or a GeoTimeSeries object");
```

Comment:
Check for null before calling poppedObject.getClass()

Reply:
Good idea, fixed in 82bc1e0